### ISSUE TITLE
Run QA Suite Image via Action - adjusted

### ISSUE THREAD

**Issue by ldraney** [department-of-veterans-affairs/notification-api#1250]
- [ ] Discuss with QA

## User Story - Business Need
As a member of the VANotify team, I need to be able to easily run the latest version of the `notification-api-qa` regression suite inside an EC2 instance running in the `utility-qa` environment, so that we can ensure the reliability and quality of our application through a comprehensive testing strategy.

### User Story
**As a** VANotify Engineer  
**I want** to be able to run the latest version of the `notification-api-qa` regression suite inside an EC2 instance running in the `utility-qa` environment  
**So that** I can ensure the reliability and quality of our application through a comprehensive testing strategy

## Engineering Checklist
- [ ] Talk with QA after picking up this ticket to sync on the work to be done.
- [ ] Configure the Github action to communicate with the EC2 instance and run the Docker image inside it using the custom action.yml file in notification-api/.github/actions/run-commands-on-ec2/action.yml
- [ ] Store the .env values in a single SSM parameter.
- [ ] Configure a GitHub action to securely pull the SSM values and echo them into a .env file on the EC2 instance.
- [ ] Ensure that the Docker run command uses the --env flag as planned, leveraging the .env file on the EC2 instance.
- [ ] Test the GitHub action to verify that the Docker container is successfully run on the EC2 instance, and the output is in the workflow.
- [ ] Document the process, and how to update the .env file, in the notification-api-qa repo's README file for future reference.

## Acceptance Criteria
- Github action communicates with the EC2 instance and runs the Docker image inside it using the custom action.yml file
- .env values are stored in a single SSM parameter
- Github action securely pulls the SSM values and echoes them into a .env file on the EC2 instance
- Docker run command uses the --env flag, leveraging the .env file on the EC2 instance
- Docker container is successfully run on the EC2 instance, and the output is in the Github action workflow
- The process and instructions for updating the .env file are documented in the notification-api-qa repo's README file

## QA Considerations
- [ ] QA Engineer has successfully run the action via dispatch
- [ ] QA approves of the documentation for running the suite via the Github Action or manually inside the EC2.

## Dependencies
- This ticket is dependent on ticket #54 "Containerized QA Repo" and ticket #55 "Upload notification-api-qa regression suite Docker image to ECR" 
- #538 should be completed first, as the EC2 instance code is not yet cleaned up and merged to master

## Potential Out of Scope Items
- Ensuring that the Docker image is built and uploaded to ECR with the correct environment variables and arguments (this is addressed in ticket #55)
- The EC2 is already up, but at the point of writing this ticket #531 is still pending and needs to be refined
